### ISSUE TITLE
Create a "Quiz Me" sprocket for general trivia

### ISSUE THREAD

**Issue by WilliamBZA** [Code52/jibbr#16]
Users should be able to ask Jibbr for a trivia type question. Jibbr will then ask a general knowledge question and keep track of users scores.

**Comment by shiftkey** [comment]
Wolfram Alpha have an API :) (free for experimental use) http://products.wolframalpha.com/api/

**Comment by WilliamBZA** [comment]
Yeah, @dfowler was using that for his original Jabbot. Very useful.

**Comment by ssboisen** [comment]
I'll be doing this if I get the time. Thinking of using True Knowledge API http://developer.trueknowledge.com. 2000 queries per day for free usage.

**Comment by shiftkey** [comment]
Cool. Can't wait to see it in action!

**Comment by ssboisen** [comment]
I submitted a pull request for the first version of this sprocket.
